operator/v1/0000_70_dns-operator_00.crd.yaml

```diff
@@ -42,6 +42,28 @@ spec:
             description: spec is the specification of the desired behavior of the
               DNS.
             properties:
+              cache:
+                description: 'cache describes the caching configuration that applies
+                  to all server blocks listed in the Corefile. This field allows a
+                  cluster admin to optionally configure: * successTTL which is a duration
+                  for which positive responses should be cached. * denialTTL which
+                  is a duration for which negative responses should be cached. If
+                  this is not configured, OpenShift will configure success and denial
+                  caching with a default value that is subject to change.'
```

Comment:
It may be subject to change, but we should probably say what the default is (I'm guessing we stick with the CoreDNS defaults[ "3600 for NOERROR responses and 1800 for denial of existence"](https://coredns.io/plugins/cache/) for the initial implementation?)

Reply:
Fixed!